submarine-experiment-prehandler/fs_prehandler/hdfs_prehandler.py

```diff
@@ -15,26 +15,43 @@
 
```

Comment:
I tried to build the docker image and found that roughly this image has 2.8G in local. 
I don't know if this will affect the overall user experience, after all, if the image is too large, it will take too much time to pull it. 
Or is it possible to reduce it to a simple java code, so that python and the whole hadoop will not be installed , and the image will contain only the jars needed inside.

Reply:
that's a good point, I take the whole hadoop deployment in it because we could access hdfs and s3 in the same approach, I could check is there any lighter way to execute distcp.

Reply:
large images size issues to be fixed in subsequent issues